src/WebJobs.Script/Binding/Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -252,6 +256,22 @@ internal static void ApplyBindingData(object target, IDictionary<string, object>
                     }
                 }
 
+                {
+                    string auth = null;
+
+                    if (request.Headers.Authorization != null)
+                    {
+                        auth = request.Headers.Authorization.Parameter;
+                    }
+                    else
+                    {
+                        // The (optional) id token from easy auth.
+                        // This can be used in a token exchange.
+                        auth = request.GetHeaderValueOrDefault("X-MS-TOKEN-AAD-ID-TOKEN"); // JWT
+                    }
+                    bindingData[AuthName] = auth ?? string.Empty;
```

Comment:
General question about binding data: if we set a property in the contract, can we leave that entry in the dictionary undefined? null?